inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/exception/ProxySdkException.java

```diff
@@ -15,22 +15,34 @@
  * limitations under the License.
  */
 
-package org.apache.inlong.sdk.dataproxy.network;
+package org.apache.inlong.sdk.dataproxy.exception;
 
-public class ProxysdkException extends Exception {
+/**
+ * Proxy Sdk Exception
+ *
+ * Used for unacceptable situations when reporting messages, such as empty input parameters,
+ * illegal parameters, abnormal execution status, and exceptions encountered during execution that
+ * were not considered during design, etc.
+ *
+ * If this exception is thrown during the debugging phase, the caller needs to check and
+ * adjust the corresponding implementation according to the exception content; if the exception
+ * is encountered during operation; the caller can try a limited number of times,
+ * and discard this report if it fails after trying again.
+ */
+public class ProxySdkException extends Exception {
```

Comment:
```suggestion
public class ProxySDKException extends Exception {
```

Reply:
Camel case naming is mainly used here. “SDK” if used, will be connected with the later "E" to form "SDKE", which makes people mistakenly think it is another special definition. At the same time, I checked and found that "SDK" and "Sdk" are both ok.